task/3scale-toolbox-import/0.1/OWNERS

```diff
@@ -0,0 +1,3 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
+owners:
+    rafamqrs
```

Comment:
```suggestion
approvers:
- rafamqrs
```

Reply:
also you need to make a PR to tektoncd/community to add yourself in the org in order to be an owner for this resource. You can take reference of the following PR https://github.com/tektoncd/community/pull/538

Reply:
done, I've created a PR to tektoncd/community 

Now, I'm waiting for the PR approval: https://github.com/tektoncd/community/pull/563